### ISSUE TITLE
Chrome/Edge version 129 breaks the drag & drop

### ISSUE THREAD

**Issue by debugHere** [mcguffin/acf-dropzone#30]
Updating to the latest version of Chrome/Edge (129) seems to break the drag and drop. Seems to overlay it over all other elements on the page (preventing accessing anything else).

Confirmed this via Lambdatest Windows 11 Chrome 128 vs 129

Using: ACF 6.3.6, ACF Dropzone 1.1.14, Wordpress 6.6.1

![Screenshot 2024-10-01 at 12 51 25](https://github.com/user-attachments/assets/4f43710a-272e-40a6-8163-db2b459b2e27)

@mcguffin

**Comment by debugHere** [comment]
bump @mcguffin

**Comment by renemgit** [comment]
Confirmed, it's useless for now...

**Comment by renemgit** [comment]
Unfortunately this update does not fix the overlay issue. I have cleared caches and Chrome still has this issue for me.

Kind regards René

**Comment by mcguffin** [comment]
Unfortuneatly I can only test in chrome for mac.
The fix was basically adding `-webkit-transform: translateZ(0);` to the container element.

What happens if you remove the vendor prefix in dev tools and just give it `transform: translateZ(0);`?  
Like this:
<img width="663" alt="grafik" src="https://github.com/user-attachments/assets/44f931be-bc47-4c45-b790-68873f936e83">

**Comment by renemgit** [comment]
**Position Fixed** seems to be the problem, right?

.dropzone .acf-gallery-attachments .acf-dropzone-info {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-orient: vertical;
    -webkit-box-direction: normal;
    -ms-flex-direction: column;
    flex-direction: column;
    height: 100%;
    -webkit-box-pack: center;
    -ms-flex-pack: center;
    justify-content: center;
    **position: fixed;**
    left: 0;
    top: 0;
    width: 100%;
}

**Comment by mcguffin** [comment]
Yes. On webkit it is `fixed` to the viewport coordinate system not the container's. 
The `transform: translateZ(0)` css rule should establish a seperate coordinate system on the container (`.dropzone .gallery-main`) which seems to work on chrome mac but not on windows.

Did you consider to answer my question above? (Changing `-webkit-transform: translateZ(0);` to `transform: translateZ(0);`)

**Comment by renemgit** [comment]
That's seem to work here on Mac and Chrome. Is this something you going to add on the plugin?

Regards René
